util/pace/util/quantity.py

```diff
@@ -489,20 +444,18 @@ def np(self) -> NumpyModule:
 
     @property
     def __array_interface__(self):
-        return self.storage.__array_interface__
+        return self.data.__array_interface__
 
     @property
     def __cuda_array_interface__(self):
-        return self.storage.__cuda_array_interface__
+        return self.data.__cuda_array_interface__
 
     @property
     def shape(self):
-        return self.storage.shape
+        return self.data.shape
 
     def __descriptor__(self):
-        if self._storage is None:
-            return None  # trigger DaCe JIT
-        return self._storage.__descriptor__()
+        return None  # trigger DaCe JIT
```

Comment:
Should get clearer on this before merging too.

Reply:
+1